hasher-matcher-actioner/hmalib/common.py

```diff
@@ -3,11 +3,24 @@
 import logging
 
 
+class PrintLogger:
+    """
+    Custom logger that just prints
+    """
+
+    def info(self, log_string):
+        print(log_string)
+
+
```

Comment:
(blocking): What's the intent of this? The logging module should be able to do this for you with the right settings.

Reply:
This was helpful for debugging as I ran locally. Instead of replacing all the `logger.info` instances with `print` I could just change the logger creation line from `logger = get_logger(__name__)` to `logger = get_logger("print")` during development

If no one else will use this I can remove. If your worried about the other logger functions could implement those as well

Reply:
Hmm. I think the issue is that i've been using `logger.info` which according to your link does not print to the console. I can switch those instances to `logger.warning` during my debug sessions. Will remove this